server/controllers/appt.controller.ts

```diff
@@ -0,0 +1,54 @@
+import { Request, Response } from "express";
+import Appointment from "../models/appt.model";
+import { IAppointment } from "../interfaces/appointment.interface";
+
+const findAllAppointments = async (req: Request, res: Response): Promise<IAppointment[]> => {
+  try {
+    const appointments = await Appointment.find()
+    return res.status(200).json(appointments)
+  } catch(e) {
+    return res.status(500).json(e.message)
+  }
+}
+
+const createNewAppointment = async (req: Request, res: Response): Promise<IAppointment> => {
```

Comment:
Potential error return type for when things fail? Would be good for all of the methods to have, this is just one that definitely needs it.

Reply:
This will not affect the functionality of the app, so we're putting it off to #31 for future enhancement.